packages/typespec-ts/test/modularUnit/scenarios/enumUnion/enumUnion.md

```diff
@@ -47,30 +47,32 @@ export type SchemaContentTypeValues =
 ## paramOutput
 
 ```ts operations
-import { DemoServiceContext as Client } from "./index.js";
+import { DemoServiceContext as Client } from "../index.js";
 import { SchemaContentTypeValues } from "../models/models.js";
 import {
   StreamableMethod,
   PathUncheckedResponse,
   createRestError,
-  operationOptionsToRequestParameters
+  operationOptionsToRequestParameters,
 } from "@azure-rest/core-client";
 
 export function _getSend(
   context: Client,
   contentType: SchemaContentTypeValues,
   body: string,
-  options: GetOptionalParams = { requestOptions: {} }
+  options: GetOptionalParams = { requestOptions: {} },
 ): StreamableMethod {
-  return context.path("/").post({
-    ...operationOptionsToRequestParameters(options),
-    contentType: contentType,
```

Comment:
need to figure out whether the contentType is parallel with body headers or should just be in headers.